packages/mui-lab/src/LoadingButton/LoadingButton.js

```diff
@@ -194,6 +196,19 @@ const LoadingButtonLoadingIndicator = styled('span', {
   ],
 }));
 
+const LoadingButtonLabel = styled('span', {
+  name: 'MuiLoadingButton',
+  slot: 'Label',
+  overridesResolver: (props, styles) => {
+    return [styles.label];
+  },
+})({
```

Comment:
Ah, I guess I forgot #43372

Reply:
We are not using it if there it is not a callback, but it is missing in the other slots, I am adding it.

Reply:
Ah right, ok, it's the opposite, makes sense.